test/WorkflowCore.Tests.MySQL/WorkflowCore.Tests.MySQL.csproj

```diff
@@ -1,9 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
-    <PackageReference Include="xunit" Version="2.4.0" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
+    <PackageReference Include="Squadron.MySql" Version="0.17.0" />
```

Comment:
@glucaci  
All of the MySQL tests are failing for me - `MySqlException - no database selected`